jitbuilder/release/cpp/samples/OperandArrayTests.cpp

```diff
@@ -466,89 +463,89 @@ OperandArrayTestMethod::testArray(TR::BytecodeBuilder *builder, bool useEqual)
    //VMOA      [0,1,2,{9,11},4,5,5,5]
    //_realArray[0,1,2,{9,11},4,5,5,5]
 
-   TR::BytecodeBuilder *change7 = OrphanBytecodeBuilder(3, (char*)"BCI_change7");
-   TR::BytecodeBuilder *nextMerge = OrphanBytecodeBuilder(4, (char*)"nextMerge");
-   TR::BytecodeBuilder *view6and7 = OrphanBytecodeBuilder(5, (char*)"BCI_view6and7");
+   BytecodeBuilder *change7 = OrphanBytecodeBuilder(3, (char*)"BCI_change7");
+   BytecodeBuilder *nextMerge = OrphanBytecodeBuilder(4, (char*)"nextMerge");
 
-   mergeBB->IfCmpEqual(change7, GET(mergeBB, 5), GET(mergeBB, 6));
-   mergeBB->AddFallThroughBuilder(nextMerge);
+   IlValue *left = GET(mergeBB, 5);
+   IlValue *right = GET(mergeBB, 6);
+   mergeBB->IfCmpEqual(change7, left, right);
 
-   nextMerge->AddFallThroughBuilder(view6and7);
+   mergeBB->AddFallThroughBuilder(nextMerge);
```

Comment:
the commit message should also mention the cleanup in this sample

Reply:
Fixing.